src/main/java/com/minecolonies/coremod/colony/requestsystem/management/manager/wrapped/WrappedStaticStateRequestManager.java

```diff
@@ -28,6 +28,6 @@ public WrappedStaticStateRequestManager(@NotNull final IStandardRequestManager w
      */
     public void updateRequestState(@NotNull final IToken token, @NotNull final RequestState state) throws IllegalArgumentException
     {
-        //TODO: implement when link is created with workers
+        //TODO: implement when link is created with workerbuildings
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)